db/blob/blob_file_reader_test.cc

```diff
@@ -277,12 +277,13 @@ TEST_F(BlobFileReaderTest, CreateReaderAndGetBlob) {
     PinnableSlice value;
     uint64_t bytes_read = 0;
 
-    ASSERT_TRUE(
-        reader
-            ->GetBlob(read_options, shorter_key,
-                      blob_offsets[0] - (keys[0].size() - sizeof(shorter_key)),
-                      blob_sizes[0], kNoCompression, &value, &bytes_read)
-            .IsCorruption());
+    ASSERT_TRUE(reader
+                    ->GetBlob(read_options, shorter_key,
+                              blob_offsets[0] -
+                                  (keys[0].size() - sizeof(shorter_key) - 1),
```

Comment:
Sorry, I meant `keys[0].size() - (sizeof(shorter_key) - 1)`  (which is equivalent to `keys[0].size() - sizeof(shorter_key) + 1`) here

Reply:
Done